Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/ManipulationHandler.cs

```diff
@@ -295,6 +294,34 @@ private Vector3 GetPointersCentroid()
             return sum / Math.Max(1, count);
         }
 
+        private MixedRealityPose GetAveragePointerPose()
+        {
+            Debug.Assert(pointerIdToPointerMap.Count > 1);
+
+            var handsEnumerator = pointerIdToPointerMap.Values.GetEnumerator();
+            handsEnumerator.MoveNext();
+            var hand1 = handsEnumerator.Current.pointer;
+            handsEnumerator.MoveNext();
+            var hand2 = handsEnumerator.Current.pointer;
+            handsEnumerator.Dispose();
+            
+            MixedRealityPose pose = new MixedRealityPose();
+            pose.Position = (hand1.Position + hand2.Position) / 2;
+
+            Vector3 right = hand1.Position - hand2.Position;
+
+            // WIP
```

Comment:
I have this marked as WIP. For one, it's doesn't work with 6dof controllers as it doesn't take the controllers rotation into account. Additionally, relying on the `rotateLogic` isn't ideal, as I'm pretty sure there are circumstances where this object will not be Setup with the correct values for the current rotation.

Reply:
This should also include the calculation of `right` above.

Reply:
Oops, forgot to remove the WIP stuff. I'll do that now.